etc/check_meta.jl

```diff
@@ -0,0 +1,33 @@
+# this can be run from the command line with:
+# > julia -e 'include("etc/check_meta.jl")'
+#
+# note: the @main function requires julia 1.11 or newer
+
+function (@main)(_)
+  issues = String[]
+  meta = """```@meta
+  CurrentModule = Oscar
+  DocTestSetup = Oscar.doctestsetup()
+  ```
+  """
+  oscardir = normpath(@__DIR__, "..")
+  for (dir, _, files) in walkdir(oscardir)
+    @show dir
+    contains(dir, "docs/src") || continue
+    for file in files
+      endswith(file, ".md") || continue
+      path = joinpath(dir, file)
+      if read(`head -n4 $path`, String) != meta
```

Comment:
Maybe do this
```suggestion
      if read(`head -n$n $path`, String) != meta
```
and then set `n` to the number of lines in `meta`?